cds-snc/cppd-medical-report-sails
# Invite code expiry fix

# What this does
Fixes a small bug - the expiry for the invite code was being displayed as +3 years rather than +3 months.

## To test
- Fill out the patient portion of the report.
- Check the expiry date on the generated code
